ui/nuxeo-document-preview.js

```diff
@@ -297,7 +297,8 @@ import './viewers/nuxeo-video-viewer.js';
           .map((conversion) => {
             return {
               data: conversion.content.data,
-              type: conversion.content['mime-type'],
+              // XXX - while backend doesn't give us a cleaner way to access codec information, we need to parse it.
```

Comment:
I think we should create a server ticket to improve this and leave a note here, in the comments, to clean this up after it is implemented.

Reply:
Ticket created: https://jira.nuxeo.com/browse/NXP-31056